domain/node_api/address_balance.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import asdict, dataclass
+from typing import Dict, Optional
+
+from dacite import from_dict
+
+
+@dataclass
+class AddressBalanceTokenData:
+    name: str
+    symbol: str
+    received: int
+    spent: int
+
+
+@dataclass
+class AddressBalance:
+
+    success: bool
+    # The next fields are optional to parse when success is False
+    tokens_data: Optional[Dict[str, AddressBalanceTokenData]] = None
```

Comment:
The key of the dict is the uid of the token?

Reply:
I think we should add a comment about it

Reply:
Done